tests/unit/acquisition/function/test_function.py

```diff
@@ -635,3 +644,59 @@ def test_batch_monte_carlo_expected_improvement_updates_without_retracing() -> N
     assert batch_ei.__call__._get_tracing_count() == 1  # type: ignore
     npt.assert_allclose(batch_ei(xs), ei(xs), rtol=0.06)
     assert batch_ei.__call__._get_tracing_count() == 1  # type: ignore
+
+
+def test_integrated_variance_reduction() -> None:
```

Comment:
these are great, but we should also have some basic tests, like building successfully an acq fnc, output shapes etc, the usual battery that you can find for other acq functions